test/plugins/validation/2and3/security-definitions-ibm.js

```diff
@@ -12,7 +13,7 @@ const config = {
 
 describe('validation plugin - semantic - security-definitions-ibm', function() {
   describe('Swagger 2', function() {
-    it('should warn about an unused security definition', function() {
+    it('should follow references to security schemes', function() {
```

Comment:
This was not the test case to change - will you switch this back and update the test you added? See my comment below